crates/wasmtime/src/runtime/vm/sys/unix/signals.rs

```diff
@@ -308,7 +308,7 @@ pub fn lazy_per_thread_init() {
     // This thread local is purely used to register a `Stack` to get deallocated
     // when the thread exists. Otherwise this function is only ever called at
     // most once per-thread.
-    thread_local! {
+    std::thread_local! {
```

Comment:
Why is `std` okay to use here? Shouldn't this use the `wasmtime_tls_{get,set}` stuff?

Reply:
I guess if we are on `unix` then we know we have `std` available?

Reply:
Yeah that was my thinking, on platforms that are known to have the standard library we go ahead and just import from the standard library. Things will get a bit messier otherwise because it's not a great experience for Wasmtime to depend on `#[repr(C)]` symbols silently so I'm hoping to keep it to a minimum.